src/Microsoft.Azure.WebJobs.Host/Bindings/BindingProviders/ItemBindingProvider.cs

```diff
@@ -16,13 +16,16 @@ internal class ItemBindingProvider<TAttribute> : IBindingProvider
     {
         private readonly INameResolver _nameResolver;
         private readonly Func<TAttribute, Type, Task<IValueBinder>> _builder;
+        private readonly OpenType _filter;
 
         public ItemBindingProvider(
                     INameResolver nameResolver,
-                    Func<TAttribute, Type, Task<IValueBinder>> builder)
+                    Func<TAttribute, Type, Task<IValueBinder>> builder,
+                    OpenType filter)
```

Comment:
It is strange to see an instance of an OpenType be called a "filter". Rename the param to openType?

Reply:
OpenTypes are filters (hence the IsMatch method). In this case, we're filtering the rule to only match on the given OpenType. I can add a comment to clarify this. 

---
In reply to: [150925434](https://github.com/Azure/azure-webjobs-sdk/pull/1422#discussion_r150925434) [](ancestors = 150925434)